src/main/webapp/style/gsn.css

```diff
@@ -824,7 +824,7 @@ input.date-picker
 
 /* gsn data part */
 div.full-page {
-    width: 730px !important;
+    width: 95% !important; /* 730px */

```

Comment:
if the #container has a width of 750px this doesn't change anything.... To take advantage of a larger page for data visualisation, it should be changed on both elements.